src/_cffi_src/openssl/ssl.py

```diff
@@ -556,6 +569,13 @@
 static const long Cryptography_HAS_NEXTPROTONEG = 0;
 static const long Cryptography_HAS_ALPN = 1;
 
+#ifdef SSL_OP_NO_RENEGOTIATION
+static const long Cryptography_HAS_OP_NO_RENEGOTIATION = 1;
+#else
+static const long Cryptography_HAS_OP_NO_RENEGOTIATION = 0;
```

Comment:
This should be used in `_conditional.py` to remove the value when it doesn't exist.